theme/horizontalrule.css

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+.ck-content .ck-horizontal-rule {
```

Comment:
This isn't a content style if it uses `.ck-horizontal-rule` because this class is not printed out by the editor. So, this is part of UI styles and as such should, AFAIK, be prepended with `.ck` instead.

cc @dkonopka

Reply:
You are completely right, thanks for being watchful 👍